### PR TITLE
[PyOV] some fixes and improvements in python tests

### PR DESCRIPTION
**Pull request by akuporos**
### Details:
 - *item1*
 - *...*

### Tickets:
 - as part of 78800

### CODE REVIEW

File: src/bindings/python/tests/test_runtime/test_core.py [hunk 1]
```diff
@@ -199,6 +199,10 @@ def test_available_devices(device):
     )
 
 
+@pytest.mark.skipif(
+    os.environ.get("TEST_DEVICE", "CPU") != "CPU",
+    reason=f"Cannot run test on device {os.environ.get('TEST_DEVICE')}, Plugin specific test",
+)
 def test_get_property():
     core = Core()
     conf = core.get_property("CPU", "CPU_BIND_THREAD")
```
Comment (ilya-lavrenov) [inline src/bindings/python/tests/test_runtime/test_core.py:208]: can we just get another property which is available on all devices? E.g. supported config keys

Comment (akuporos) [reply src/bindings/python/tests/test_runtime/test_core.py]: yes, done

**Review by ilya-lavrenov — APPROVED** [review]

File: src/bindings/python/tests/test_graph/test_preprocess.py [hunk 2]
```diff
@@ -594,7 +594,7 @@ def test_ngraph_set_layout_by_str_thow_exception(layout):
 
     with pytest.raises(RuntimeError) as e:
         ppp.input().model().set_layout(layout)
-    assert "Layout name is invalid" in str(e)
+        assert "Layout name is invalid" in str(e)
```
Comment (jiwaszki) [inline src/bindings/python/tests/test_graph/test_preprocess.py:597]: Why there is an additional indentation? I do not recall it from pytest docs. There is always assert outside the context manager.

File: src/bindings/python/tests/test_graph/test_preprocess.py [hunk 3]
```diff
@@ -594,7 +594,7 @@ def test_ngraph_set_layout_by_str_thow_exception(layout):
 
     with pytest.raises(RuntimeError) as e:
```
Comment (jiwaszki) [inline src/bindings/python/tests/test_graph/test_preprocess.py:595]: If you want to improve asserts here, I would try something like this to remove `assert`:
```suggestion
    with pytest.raises(RuntimeError, match=r"Layout name is invalid"):
```

Comment (akuporos) [reply src/bindings/python/tests/test_graph/test_preprocess.py:595]: I'd prefer to keep assert

File: src/bindings/python/tests/test_graph/test_preprocess.py (hunk 2 above)
Comment (akuporos) [reply src/bindings/python/tests/test_graph/test_preprocess.py]: discussed offline

**Review by jiwaszki — APPROVED** [review]

**Pull request merged**